kernel/ps2/src/lib.rs

```diff
@@ -207,168 +206,53 @@ fn write_data(value: WritableData) {
 // wiki.osdev.org/%228042%22_PS/2_Controller#PS.2F2_Controller_Configuration_Byte
 /// Used for [HostToControllerCommand::ReadFromInternalRAMByte0] and [HostToControllerCommand::WriteToInternalRAMByte0]
 #[bitfield(bits = 8)]
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
this (and probably all the other bitfield types) can also derive `Copy`, since they're just plain old data types.